astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+---
+title: .Net
+description: Quickstart integration of .Net web application with FusionAuth
```

Comment:
Long story.  It is essentially the bourbon is whiskey, but whiskey is not bourbon.  So I think most specifically this can be labeled ASP.NET Core. It can also be.NET Core or .NET and all are accurate.  I think .NET just has a broader reach.

Reply:
Sounds good.